gerblook/gerber.py

```diff
@@ -61,6 +63,31 @@ def guess_layer(path, gerberdir):
     if 'npth' in filename:
       return 'nonplated_drills'
     return 'plated_drills'
+
+  # KiCad default naming
+  match = re.search(r'-([^-.]+)\.([^.]+)\.gbr', filename)
```

Comment:
Older KiCad uses an underscore (`_`)  in the layer names instead of a dot (`.`). 

Also, check out the [test cases](https://github.com/tracespace/whats-that-gerber/blob/master/test/filenames-by-cad.js) we have come up with for whats-that-gerber.